### ISSUE TITLE
Additional CSS line count should be narrower

### ISSUE THREAD

**Issue by melchoyce** [WordPress/better-code-editing#83]
Despite only having one digit per line, the line count is quite wide:

<img src="https://user-images.githubusercontent.com/2846578/30081738-600b32b8-9256-11e7-9812-74810b2710e8.png" width="300" />

**Comment by westonruter** [comment]
@melchoyce the gutter width is to give room for the error/warning icons.

**Comment by melchoyce** [comment]
Does it need that much space? And are the theme/plugin editors different because the php files aren't being linted yet?
 
![image](https://user-images.githubusercontent.com/2846578/30089025-1304abe4-9278-11e7-8d83-aa21b681387b.png)

**Comment by westonruter** [comment]
@melchoyce yes, that's right. There isn't a linter for PHP yet (#48) so the gutter is not present.

**Comment by melchoyce** [comment]
Okay 👍 

Any way we could still tighten up the space a bit?

**Comment by westonruter** [comment]
Probably, but it would require some CodeMirror research. Help wanted.

**Comment by kenashworth** [comment]
CodeMirror takes the number columns width right from the CSS, so adding `.CodeMirror-linenumber { min-width: 1px; }` would shrink it as much as is possible (without clipping the numbers). The column will grow with number of digits in the largest line number automatically. 

(Additionally you can shave off a few more pixels with padding on that class. `.CodeMirror-linenumber { min-width: 1px; padding: 0 1px; }`)

The biggest space is the 16px for the linting and hinting, which I haven't yet figured out how to shrink. Maybe there's a way to overlay the error icon onto the numbers? Worth looking into, or nah?
